index.js

```diff
@@ -498,10 +498,18 @@ prototype.buildOperation = function(oldOperation, operationDefaults) {
     parameters.push(this.buildParameter(oldParameter));
   });
 
+  var tags = (operationDefaults.tags || []).concat(oldOperation.tags || []);
+
+  tags = tags.filter(function(e, i, arr) {
```

Comment:
Oh I see. You're returning the boolean. Can you change the variable name or simply return `arr.lastIndexOf(e) === i`?

Tags are strings at this point, right?